scripts/soltest.sh

```diff
@@ -64,4 +64,4 @@ if [ "$USE_DEBUGGER" -ne "0" ]; then
 	DEBUG_PREFIX=${DEBUGGER}
 fi
 
-exec ${DEBUG_PREFIX} ${SOLIDITY_BUILD_DIR}/test/soltest ${BOOST_OPTIONS} -- --testpath ${REPO_ROOT}/test ${SOLTEST_OPTIONS}
+exec ${DEBUG_PREFIX} "${SOLIDITY_BUILD_DIR}/test/soltest" ${BOOST_OPTIONS} -- --testpath "${REPO_ROOT}/test" ${SOLTEST_OPTIONS}
```

Comment:
Why no quote around `DEBUG_PREFIX`? Actually I'm curious if this script passes the shell checker and if it does why the style checker did not warn on the missing quotes around the said variable.

Reply:
shellcheck does warn here. This file is still on the ignore list because it has other shellcheck issues and this PR fixes only quoting.

The problem here is that `$DEBUG_PREFIX` variable is unset unless you use the `--debugger` option. If I quoted it, `exec` would start getting an empty string as the command name when the variable is unset. As it is now, the argument is simply ignored when empty and the next parameter becomes the command name.

That's another case that will be fixed separately.

Reply:
Ah I see, I overlooked the fact that this file is still in the ignore list. I assumed this PR removes the said file from the ignore file.